x/liquiditypool/keeper/msg_server_position.go

```diff
@@ -226,8 +214,15 @@ func (k Keeper) DecreaseLiquidity(ctx sdk.Context, sender sdk.AccAddress, positi
 		return math.Int{}, math.Int{}, errorsmod.Wrapf(types.ErrPoolNotFound, "pool id: %d", position.PoolId)
 	}
 
-	liquiditDelta := liquidity.Neg()
-	amountBase, amountQuote, lowerTickEmpty, upperTickEmpty, err := k.UpdatePosition(ctx, position.PoolId, sender, position.LowerTick, position.UpperTick, liquiditDelta, positionId)
+	if liquidity.Equal(position.Liquidity) {
```

Comment:
Question:

Is `if liquidity.Equal(position.Liquidity) {` really needed?
I am thinking of a possibility that we should collect fees anytime

Reply:
Need to dig the relation with `k.SetAccumulatorPositionFeeAccumulator`

Reply:
I don't see a problem since this function is only called for DecreaseLiquidity and IncreaseLiquidity.
simple and good.

Reply:
I've already merged #141, and to avoid confusion, it's better to change the spec from v0.3.x.